src/py/flwr/cli/new/templates/app/code/server.xgboost.py.tpl

```diff
@@ -0,0 +1,54 @@
+"""$project_name: A Flower / XGBoost app."""
+
+from typing import Dict
+
+from flwr.common import Context
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server.strategy import FedXgbBagging
+
+
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    total_num = sum([num for num, _ in eval_metrics])
+    auc_aggregated = (
+        sum([metrics["AUC"] * num for num, metrics in eval_metrics]) / total_num
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
+    return metrics_aggregated
+
+
+def config_func(rnd: int) -> Dict[str, str]:
+    """Return a configuration with global epochs."""
+    config = {
+        "global_round": str(rnd),
+    }
+    return config
+
+
+def server_fn(context: Context):
+    # Read from config
+    num_rounds = int(context.run_config["num-server-rounds"])
+    pool_size = int(context.run_config["pool-size"])
+    num_clients_per_round = int(context.run_config["num-clients-per-round"])
+    num_evaluate_clients = int(context.run_config["num-evaluate-clients"])
+
+    # Define strategy
+    strategy = FedXgbBagging(
+        fraction_fit=(float(num_clients_per_round) / pool_size),
```

Comment:
this looks a bit strange. @yan-gao-GY couldn't we just hardcode it to let's say `0.5`? We shouldn't have a variable called `num_clients_per_round` if then the number of actual clients sampled is not the one set by the variable.